README.md

```diff
@@ -124,30 +128,29 @@ In the **desktop application**, open File > Import > MD and select your Markdown
 
 In the **terminal application**, in [command-line mode](https://github.com/laurent22/joplin/blob/master/readme/terminal.md#command-line-mode), type `import --format md /path/to/file.md` or `import --format md /path/to/directory/`.
 
-## Importing from other applications
+## Importing in Linux and other miscellaneous applications
```

Comment:
Linux specific info does not have a dedicated section yet, so incorporating it here until a Linux section is created made sense given the Linux apps mentioned.

Reply:
Please don't change the title has it might be linked to from somewhere else. Also it's not just about Linux.